package.json

```diff
@@ -12,9 +12,23 @@
 	},
 	"dependencies": {
 		"@react-native-async-storage/async-storage": "^1.15.8",
+		"@react-navigation/native": "^6.0.4",
+		"@react-navigation/stack": "^6.0.9",
 		"@tensorflow-models/coco-ssd": "^2.2.2",
 		"@tensorflow/tfjs": "^3.9.0",
 		"@tensorflow/tfjs-react-native": "^0.7.0",
+		"@types/istanbul-lib-report": "^3.0.0",
+		"@types/istanbul-reports": "^3.0.1",
+		"@types/json-schema": "^7.0.9",
+		"@types/node-fetch": "^3.0.3",
+		"@types/offscreencanvas": "^2019.6.4",
+		"@types/prop-types": "^15.7.4",
+		"@types/scheduler": "^0.16.2",
+		"@types/seedrandom": "^3.0.1",
+		"@types/stack-utils": "^2.0.1",
+		"@types/webgl2": "^0.0.6",
+		"@types/yargs": "^17.0.3",
+		"@types/yargs-parser": "^20.2.1",
```

Comment:
I think all of these `@types` packages can go in `devDependencies`. Also, I don't think we need `@types/prop-types`, since we are using typescript.